### DISCUSSION TITLE
6.3.1 (2022-12-15)

### DISCUSSION THREAD

**Discussion by manusa** [fabric8io/kubernetes-client#4685] [Announcements]
#### Bugs
- Fix #4666: fixed okhttp calls not explicitly closing
- Fix #4673: fixes a regression in sharing the OpenShiftOAuthInterceptor token
- Fix #4677: [java-generator] Fix default encoding of enums

**Full Changelog**: https://github.com/fabric8io/kubernetes-client/compare/v6.3.0...v6.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/fabric8io/kubernetes-client/releases/tag/v6.3.1'>6.3.1 (2022-12-15)</a>.</em>

**Comment by scholzj** [comment]
Thanks a lot @shawkins and @manusa ... I do not see any unclosed connections in OkHttp with 6.3.1! 😍
